clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -5910,6 +5910,30 @@ static void handleBuiltinAliasAttr(Sema &S, Decl *D,
   D->addAttr(::new (S.Context) BuiltinAliasAttr(S.Context, AL, Ident));
 }
 
+static void handleDebugInfoTypeAttr(Sema &S, Decl *D, const ParsedAttr &AL) {
+  if (!AL.hasParsedType()) {
+    S.Diag(AL.getLoc(), diag::err_attribute_wrong_number_arguments) << AL << 1;
+    return;
+  }
+
+  TypeSourceInfo *ParmTSI = nullptr;
+  QualType type = S.GetTypeFromParser(AL.getTypeArg(), &ParmTSI);
```

Comment:
```suggestion
  QualType QT = S.GetTypeFromParser(AL.getTypeArg(), &ParmTSI);
```
(or any other name that meets the coding style, except `Type` because.. that's a type.)

Reply:
Fixed